spec/services/change_schedule_spec.rb

```diff
@@ -199,6 +199,32 @@
                 service.call
               }.to change { participant_profile.reload.school_cohort }.to(new_school_cohort)
             end
+
```

Comment:
could we add more test scenarios please, to cover the different cases.

First if there is an existing partnership vs new partnership, CIP vs FIP, induction records with cohort change vs no cohort change, existing school cohort vs non existing etc.